config_distro.py

```diff
@@ -45,7 +45,7 @@
 
 SQLITE_PATH = os.path.join(DATA_DIR, 'config', 'pgadmin4.db')
 
-SESSION_DB_PATH = '/dev/shm/pgAdmin4_session'
+SESSION_DB_PATH = os.path.join(DATA_DIR, 'sessions')
```

Comment:
Wondering if there should be a separate `ENV` var to configure this; changing this path to be inside DATA_DIR will also result in sessions being persisted when restarting the container, which wouldn't be the case if it's stored in `/dev/shm`.

wdyt?

Reply:
Thanks for the review, and good point, I'll take another crack at it :)